beetsplug/subsonicupdate.py

```diff
@@ -78,10 +86,26 @@ def loaded(self):
             'v': '1.15.0',  # Subsonic 6.1 and newer.
             'c': 'beets'
         }
+
+        if port == 443:
+            schema = 'https://'
```

Comment:
I actually think we can omit the automatic scheme switching based on the port altogether if we have the URL version… the idea being that, if you want to use https, you _must_ use the new-style configuration.